File: Excercise_Files/04_javaScript/09_avgList.js
// this program calculates the average of the items of an array

// declare array
let arrayToAvg = [20, 30, 25, 35, -16, 60, -100];
// count and store number of items in array
let arrayToAvgLen = arrayToAvg.length;
// declare and initiate sum
let sum = 0;

// iterate through array and sum all items
for (let i = 0; i < arrayToAvgLen; i++) {
  sum += arrayToAvg[i];
}

// declare and calculate average
let avg = sum / arrayToAvgLen;

// print average
console.log('the average of array is: ' + avg);

File: Excercise_Files/02_Python/09_avgList.py
# this program calculates the average of all values of a list

# declare list
listToAvg = [20, 30, 25, 35, -16, 60, -100]

# declare variable sum
sum = 0
# iterate through list and sum all elements
for x in listToAvg:
  sum += x
# declare variable avg and calculate average of list
avg = sum / len(listToAvg)
# print result
print('Average value of the list elements is:', avg)
